tests/tests_basics_forwards_cacert.yml

```diff
@@ -90,8 +90,26 @@
       register: __result
       failed_when: "'error' in __result.stdout or __result is failed"
 
+    - name: Generated a file to check severity_and_facility
+      copy:
+        dest: /tmp/__testfile__
+        content: |
+          # Ansible managed
+          ruleset(name="forwards_severity_and_facility") {
+              local1.info action(name="forwards_severity_and_facility"
+                  type="omfwd"
+                  Target="host.domain"
+                  Port="1514"
+                  Protocol="tcp"
+                  StreamDriver="gtls"
+                  StreamDriverMode="1"
+                  StreamDriverAuthMode="anon"
+                  StreamDriverPermittedPeers="['*.virt.pnr.lab.eng.rdu2.redhat.com']"
```

Comment:
please use a test domain like `*.example.com`

Reply:
please squash db9cf197e27f404b4c05a8847bea5da60974f129 with the previous commit.